scripts/linux/trcE2ETest.sh

```diff
@@ -866,10 +867,12 @@ quickstart_working_folder="$working_folder/quickstart"
 image_architecture_label=$(get_image_architecture_label)
 optimize_for_performance=true
 log_upload_enabled=true
+log_rotation_max_file="125"
```

Comment:
125 files * 4MB each * ~16 modules = ~8GB of disk. 

Per https://docs.microsoft.com/en-us/azure/virtual-machines/dv3-dsv3-series, Standard_D4s_v3 support up to 32GB disk. I think we should be safe to bump the disk usage up to ~30GB i.e. 480 files?

Reply:
Some test agents don't have nearly that amount of space. It might be safer to have some agents fail due to disk space so we can see longer logs some of the time. We should handle in a separate PR.

Reply:
When I checked some agents only had ~6 GB of free space, so I thought this was already pushing it. But we can adjust further per logic above.